settings/pkg/service/v0/service.go

```diff
@@ -32,12 +33,21 @@ type Service struct {
 // NewService returns a service implementation for Service.
 func NewService(cfg *config.Config, logger log.Logger) Service {
 	service := Service{
-		id:      "ocis-settings",
-		config:  cfg,
-		logger:  logger,
-		manager: store.New(cfg),
+		id:     "ocis-settings",
+		config: cfg,
+		logger: logger,
+	}
+
+	switch cfg.StoreType {
+	default:
+		fallthrough
+	case "metadata":
+		service.manager = metastore.New(cfg)
+	case "filesystem":
+		service.manager = filestore.New(cfg)
+		// TODO: if we want to further support filesystem store it should use default permissions from store/defaults/defaults.go instead using this duplicate
+		service.RegisterDefaultRoles()
```

Comment:
I don't see any roles stored in the metadata service. Shouldn't we call RegisterDefaultRoles in all cases?

Reply:
No. Metadata store package is setting the default roles form `store/defaults/defaults.go` This is a duplicate.